src/templates/static/about/group_history.jsx

```diff
@@ -39,6 +39,7 @@ const GroupHistory = () => (
             <div className='timeline'>
                 <UL items={[
                     { className: 'year center-text',                                                           text: '2019' },
+                    { className: 'event featured',          header: it.L('Deriv.com launched'),             text: it.L('[_2] was launched as the next-generation online trading platform. [_1] and Deriv collectively handle over 200 million transactions, with over USD 1 billion in turnover a year.', it.website_name, '<a target=\'_blank\' href=\'https://deriv.com\' rel=\'noopener noreferrer\'>Deriv.com</a>') },
```

Comment:
@sash-binary can you fix the indentation of header and text so they match the other lines please?

Reply:
updated